core/src/test/scala/unit/kafka/server/AbstractFetcherThreadTest.scala

```diff
@@ -82,7 +82,7 @@ class AbstractFetcherThreadTest {
 
     // add one partition to create the consumer lag metric
     fetcher.setReplicaState(partition, MockFetcherThread.PartitionState(leaderEpoch = 0))
-    fetcher.addPartitions(Map(partition -> initialFetchState(0L, leaderEpoch = 0)))
+    fetcher.addPartitions(Map(partition -> initialFetchState(Some(topicIds.get(partition.topic())),0L, leaderEpoch = 0)))
```

Comment:
nit: If we keep `topicIds` as a Scala map, we would get an Option directly with `topicIds.get(...)`. Also, a space is missing after the `,`.